WeakAurasOptions/BuffTrigger2.lua

```diff
@@ -189,7 +196,9 @@ local function CreateNameOptions(aura_options, data, trigger, size, isExactSpell
         WeakAuras.UpdateThumbnail(data)
         WeakAuras.ClearAndUpdateOptions(data.id)
       end,
-      validate = isExactSpellId and WeakAuras.ValidateNumeric or nil
+      validate = isExactSpellId and WeakAuras.ValidateNumeric or nil,
+      control = "WeakAurasInputFocus",
+      getWithFocus = function() return trigger[optionKey] and trigger[optionKey][i] or "" end
```

Comment:
I'm not sure if that works with multi-selection.

Reply:
It does not error, here is a video of the behavior with 2 auras with aura trigger

https://github.com/WeakAuras/WeakAuras2/assets/189656/22d88b43-495e-4ed7-a7b8-6db4f02e48bf